docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -1,204 +1,309 @@
-roles: "kube,app,discovery"
-joinParams:
-  method: "token"
-  tokenName: "${TOKEN}"
-kubeClusterName: "${CLUSTER}"
-enterprise: true
-EOF
-# Install the helm chart with the values.yaml setting
-$ helm install teleport-agent teleport/teleport-kube-agent \
-  -f values.yaml \
-  --create-namespace \
-  --namespace=teleport-agent \
-  --version (=teleport.version=)
-```
-
-</TabItem>
-<TabItem scope={["cloud"]} label="Teleport Cloud">
-
-Switch `kubectl` to the Kubernetes cluster `cookie` and run the following
-commands, assigning `PROXY_ADDR` to the address of your Teleport Cloud tenant.
-
-```code
-$ PROXY_ADDR=<Var name="mytenant.teleport.sh" />:443
-
-# Install Kubernetes agent. It dials back to the Teleport cluster at $PROXY_ADDR
-$ CLUSTER=<Var name="cookie" />
-# Create the values.yaml file
-$ cat > values.yaml << EOF
-authToken: "${TOKEN}"
-proxyAddr: "${PROXY_ADDR}"
-roles: "kube,app,discovery"
-joinParams:
-  method: "token"
-  tokenName: "${TOKEN}"
-kubeClusterName: "${CLUSTER}"
-enterprise: true
-EOF
-# Run the helm install specifying to match to the Teleport Cloud version of Teleport
-$ helm install teleport-agent teleport/teleport-kube-agent \
-  -f values.yaml \
-  --create-namespace \
-  --namespace=teleport-agent \
-  --version (=cloud.version=)
-```
-
-</TabItem>
-</Tabs>
-
-Make sure that the Teleport agent pod is running. You should see one Teleport
-agent pod pod with a single ready container:
-
-```code
-$ kubectl -n teleport-agent get pods
-NAME               READY   STATUS    RESTARTS   AGE
-teleport-agent-0   1/1     Running   0          32s
-```
-
-## Step 3/3 Access your Kubernetes cluster
-
-(!docs/pages/includes/kubernetes-access/rbac.mdx!)
-
-### View pods in your cluster
-
-List connected clusters using `tsh kube ls` and switch between
-them using `tsh kube login`:
-
-```code
-$ tsh kube ls
-
-# Kube Cluster Name Selected
-# ----------------- --------
-# cookie
-
-# kubeconfig now points to the cookie cluster
-$ tsh kube login cookie
-# Logged into kubernetes cluster "cookie". Try 'kubectl version' to test the connection.
-
-# kubectl command executed on `cookie` but is routed through the Teleport cluster.
-$ kubectl get pods
-```
-
-## Next Steps
-
-- Take a look at a [kube-agent helm chart reference](../reference/helm-reference/teleport-kube-agent.mdx) for a full list of parameters.
+## Step 1/4. Select guided or manual enrollment
+
+There are two options for enrolling a Kubernetes cluster as a resource:
+
+- You can follow the guided enrollment steps in the Teleport Web UI.
+- You run commands and edit files manually in a terminal.
+
+The guided enrollment simplifies the deployment process by pre-populating commands
+and files with required information—for example, the token used to provision the
+Kubernetes cluster—and information you specify, such as the Kubernetes namespace,
+users, and groups to grant access to.
+
+If you can't use the Teleport Web UI or want to run commands and edit file manually,
```

Comment:
When would a user be unable to use the Web UI? If there aren't any situations where this would be impossible, would it make sense to use only the guided enrollment step in this guide? 

Since this is a getting started guide, I think it would make sense to remove Step 1 and "Enroll Kubernetes manually", and have all readers follow Step 2. Then we can use the "Next steps" section to link the reader to the [Kubernetes Discovery](/docs/kubernetes-access/discovery/) guides or [Registering Kubernetes Clusters with Teleport](goteleport.com/docs/kubernetes-access/register-clusters/) for their more advanced use cases.

How does that sound?